web/src/main/java/org/jivesoftware/openfire/plugin/ofmeet/InterfaceConfigServlet.java

```diff
@@ -45,6 +45,9 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
         {
             Log.trace( "[{}] interface_config requested.", request.getRemoteAddr() );
 
+            response.setCharacterEncoding( "UTF-8" ); 
+            response.setContentType( "UTF-8" );
```

Comment:
Should this be `response.setContentType( "text/html );` ?!?

Reply:
It should be "application/javascript"

Reply:
Sounds even better ;)